packages/shared/src/utils/phone.ts

```diff
@@ -1,12 +1,41 @@
-import { parsePhoneNumberWithError } from 'libphonenumber-js';
+import { parsePhoneNumberWithError } from 'libphonenumber-js/mobile';
+import type { E164Number, PhoneNumber } from 'libphonenumber-js/mobile';
+
+export { ParseError } from 'libphonenumber-js/mobile';
+
+function validateE164Number(value: string): asserts value is E164Number {
+  if (value && !value.startsWith('+')) {
+    throw new TypeError(`Invalid E164Number: ${value}`);
+  }
+}
+
+const parseE164Number = (value: string): E164Number | '' => {
+  if (!value) {
+    /**
+     * The type inference engine can not properly infer the type of the empty string,
+     * but using `string` instead. So we need to cast it.
+     */
+    // eslint-disable-next-line no-restricted-syntax
+    return value as '';
+  }
+
+  const result = value.startsWith('+') ? value : `+${value}`;
```

Comment:
The raw values start with + are not allowed in our DB?

Reply:
See packages/shared/src/utils/phone.ts change, this is a legacy logic. But we refactored this.